pages/_app.js

```diff
@@ -2,7 +2,12 @@ import GlobalStyle from "../styles";
 import Head from "next/head";
 import { useRouter } from "next/router";
 import { useState } from "react";
-import { useProjectsList } from "../components/custom-hooks/useProjectsList";
+// import useRestructureProject from "../components/custom-hooks/useRestructureProject";
+import { SWRConfig } from "swr";
+import useSWR from "swr";
+import dayjs from "dayjs";
```

Comment:
Are you utilizing dayjs here ?

Reply:
yes in line 47 and 48

Reply:
Oh apologies it slipped from my sight 😵‍💫